src/components/inline-inputs/inline-inputs-test.stories.tsx

```diff
@@ -46,3 +46,22 @@ export const Default = (props: InlineInputsProps) => {
 };
 
 Default.storyName = "default";
+
+export const RequiredOptional = () => {
```

Comment:
**suggestion** (non-blocking): I don't think there's a need to add a new story for these - we already have stories in the `inline-inputs.stories.tsx` for both `required` and `optional` cases. While they do currently have Chromatic disabled, I'd rather just remove that to enable Chromatic on them than to add new stories (where we're having to explicitly override the `disableSnapshot: true` anyway)

Reply:
Oh I must've missed them otherwise I would've just enabled the Chromatic there 😅 I'll get it sorted